src/core/binding.c

```diff
@@ -166,6 +166,7 @@ QuicBindingInitialize(
 
     *NewBinding = Binding;
     Status = QUIC_STATUS_SUCCESS;
+    InterlockedIncrement16(&Binding->Initialized);
```

Comment:
I believe a simpler solution would be to just use `RefCount` instead. It's not incremented until the library has accepted the binding (after initialize). So, down below just check if `RefCount > 0`.

Reply:
The ref count is initialized slightly later, but that should be fine to use as well.